src/test/groovy/com/stormpath/tck/util/TestAccount.groovy

```diff
@@ -16,23 +16,31 @@ class TestAccount {
     final String username = "foo-" + randomUUID
     String href
 
-    def TestAccount() {
-        def account = [email: email,
-            password: password,
-            givenName: givenName,
-            surname: surname,
-            middleName: middleName,
-            username: username]
+    public static TestAccount create() {
```

Comment:
@edjiang I made some changes to `TestAccount` that I want to flag with you -- moved the actual (API) creation into a static `create` method. That way, you can do:

- `new TestAccount()` to initialize the class with test data (but don't create anything on the server)
- `TestAccount.create()` to both initialize the test data and create it on the server.

See RegisterIT line 232 for a test that uses the former instead of the latter.

I suppose it could also be implemented as an instance method on `TestAccount` instead of a static method. Let me know if you have any preference.